google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -110,7 +110,8 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
   public enum PublicAccessPrevention {
     ENFORCED("enforced"),
     /** Default value for Public Access Prevention */
-    UNSPECIFIED("unspecified");
+    UNSPECIFIED("unspecified"),
+    UNKNOWN(null);
```

Comment:
Let's add a comment to indicate that this will be returned if we receive an unrecognized value from the API.

Reply:
fixed